examples/minelogy.py

```diff
@@ -12,95 +14,88 @@ def __init__(self, item_list, items_to_craft, mcrecipes):
-                  ),
-                 ({'blocks': [{'type': 'gravel'}],
-                   'tools': [None]},
-                  {'type': 'gravel'}
-                  ),
-                 ({'blocks': [{'type': 'sandstone'}],
-                   'tools': ['iron_pickaxe', 'stone_pickaxe', 'wooden_pickaxe']},
-                  {'type': 'sandstone'}
-                  ),
-                 ({'blocks': [{'type': 'stone'}],
-                   'tools': ['iron_pickaxe', 'stone_pickaxe', 'wooden_pickaxe']},
-                  {'type': 'stone'}
-                  ),
-                 ({'blocks': [{'type': 'deepslate'}],
-                   'tools': ['iron_pickaxe', 'stone_pickaxe', 'wooden_pickaxe']},
-                  {'type': 'deepslate'}
-                  ),
-                 ({'blocks': [{'type': 'stone', 'variant': 'stone'}],
-                   'tools': ['iron_pickaxe', 'stone_pickaxe', 'wooden_pickaxe']},
-                  {'type': 'cobblestone'}
-                  ),
-                 ({'blocks': [{'type': 'cobblestone'}],
-                   'tools': ['iron_pickaxe', 'stone_pickaxe', 'wooden_pickaxe']},
-                  {'type': 'cobblestone'}
-                  ),
-                 ({'blocks': [{'type': 'coal_ore'}],
-                   'tools': ['iron_pickaxe', 'stone_pickaxe', 'wooden_pickaxe']},
-                  {'type': 'coal'}
-                  ),
-                 ({'blocks': [{'type': 'diamond_ore', 'depthmin': 5}],
-                   'tools': ['iron_pickaxe']},
-                  {'type': 'diamond'}
-                  ),
-                 ({'blocks': [{'type': 'copper_ore'}],
-                   'tools': ['iron_pickaxe', 'stone_pickaxe']},
-                  {'type': 'raw_copper'}
-                  ),
-                 ({'blocks': [{'type': 'iron_ore', 'depthmin': 25}],
-                   'tools': ['iron_pickaxe', 'stone_pickaxe']},
-                  {'type': 'raw_iron'}
-                  ),
-                 ({'blocks': [{'type': 'pumpkin'}],
-                   'tools': [None]},
-                  {'type': 'pumpkin'}
-                  ),
-                 ({'blocks': [{'type': 'leaves', 'variant': 'oak'}],
-                   'tools': [None]},
-                  {'type': 'apple'}
-                  ),
-                 ({'blocks': [{'type': 'leaves'}],
-                   'tools': [None]},
-                  {'type': 'sapling'}
-                  ),
-                 ({'blocks': [{'type': 'tallgrass'}],
-                   'tools': [None]},
-                  {'type': 'wheat_seeds'}
-                  ),
-                 ({'blocks': [{'type': 'deadbush'}],
-                   'tools': [None]},
-                  {'type': 'stick'}
-                  ),
-                 ({'blocks': [{'type': 'tallgrass'}],
-                   'tools': [None]},
-                  {'type': 'wheat_seeds'}
-                  ),
-                 ]
+        self.mines = []
+        # self.mines = [({'blocks': [{'type': 'log'}],
```

Comment:
i would rather keep only a small part of this code block. A couple of tool-block examples would be enough for documentation purposes.

Reply:
@noskill  done